tsconfig.json

```diff
@@ -31,7 +31,7 @@
     "importHelpers": true,
     "target": "es2017",
     "module": "es2020",
-    "lib": ["es2019", "dom"]
+    "lib": ["es2018", "dom"]
```

Comment:
Why the downgrade here?

Reply:
I needed it for flatMap. But I decided against it and chose the default instead for compatibility.